### PR TITLE
Update setup.py to adapt repo change

### PR DESCRIPTION
**Pull request by yinweisu**
* Updated setup.py with necessary util functions, and modified them to adapt repo change. Installation has been tested locally.
* Updated dependency version ranges

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Train and deploy AutoGluon backed models on the cloud.
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (gradientsky) [inline NOTICE:2]: 2022

File: src/autogluon/cloud/version.py [hunk 2]
```diff
@@ -1,2 +0,0 @@
-"""This is the autogluon version file."""
-__version__ = '0.1b20221111'
```
Comment (gradientsky) [inline src/autogluon/cloud/version.py:2]: This file should be in `.gitignore` because it is auto-generated.

File: setup.py [hunk 3]
```diff
@@ -1,50 +1,134 @@
...
 install_requires = [
-    # version ranges added in ag.get_dependency_version_ranges()
-    "autogluon.common<0.7",
-    "boto3",
-    "numpy",
-    "packaging<22",
-    "pandas",
+    "autogluon.common>=0.6",
+    "boto3<2.0",
+    "numpy>=1.21,<2.0",
```
Comment (gradientsky) [inline setup.py:109]: Security: `numpy>=1.21.4`

**Comment by gradientsky** [comment]
`CONTRIBUTING` is empty - should be updated since it have important sections about security reporting.

File: src/autogluon/cloud/version.py (hunk 2 above)
Comment (yinweisu) [reply src/autogluon/cloud/version.py:2]: Yea, this is deleting it. I've added it to `.gitignore`

**Comment by yinweisu** [comment]
@gradientsky Will update `CONTRIBUTING.md` in follow-up PRs. Regarding security reporting, should I just reuse what's in `autogluon` `CONTRIBUTING.md`?

**Review by gradientsky — APPROVED** [review]

**Pull request merged**